.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build documentation
+    if: ${{ secrets.ENCRYPTION_KEY }}
```

Comment:
Ah it seems that we can't do conditionals on secrets... One way to do this is to move the the `env` L32 to the build workflow and then check the `env` var